lib/utils/ember.js

```diff
@@ -336,12 +331,50 @@ function isInjectedServiceProp(node, importedEmberName, importedInjectName) {
   );
 }
 
-function isInjectedControllerProp(node) {
-  return isPropOfType(node, 'controller');
+/**
+ * Checks if a node is a controller injection. Looks for:
+ * * controller()
```

Comment:
Looks like this is missing an import check of the imported `controller` name (we could address as a follow-up).

Reply:
Good point - will add these separately as follow ups

Reply:
@bmish Where is `controller` imported from? I couldn't find existing uses of this 😅

Reply:
```js
import Controller from '@ember/controller';
```

Reply:
Thank you!